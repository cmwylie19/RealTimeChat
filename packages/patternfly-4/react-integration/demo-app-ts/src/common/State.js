"use strict";
exports.__esModule = true;
var State = /** @class */ (function () {
    function State(name, abbreviation, capital, founded) {
        var _this = this;
        this.toString = function () { return _this.name + " (" + _this.abbreviation + ") - Founded: " + _this.founded; };
        this.name = name;
        this.abbreviation = abbreviation;
        this.capital = capital;
        this.founded = founded;
    }
    return State;
}());
exports.State = State;
