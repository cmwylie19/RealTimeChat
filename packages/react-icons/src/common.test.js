import { IconSize, getSize } from './common';

Object.values(IconSize).forEach(size => {
  test(`getSize returns size for ${size} icons`, () => {
    expect(getSize(size)).toMatchSnapshot();
  });
});

test('getSize returns a default size for unknown sizes', () => {
  expect(getSize('unknown')).toMatchSnapshot();
});
