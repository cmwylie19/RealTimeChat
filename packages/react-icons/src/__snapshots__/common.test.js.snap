// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getSize returns a default size for unknown sizes 1`] = `"1em"`;

exports[`getSize returns size for lg icons 1`] = `"2em"`;

exports[`getSize returns size for md icons 1`] = `"1.5em"`;

exports[`getSize returns size for sm icons 1`] = `"1em"`;

exports[`getSize returns size for xl icons 1`] = `"3em"`;
